lib/games_collection.rb

```diff
@@ -12,4 +12,16 @@ def generate_objects_from_csv(csv_games_path)
     end
     objects
   end
+
+  def highest_total_score
+    @games.map do |game|
+      game.away_goals.to_i + game.home_goals.to_i
```

Comment:
This line and line 24 could be broken out as a helper method - and I wrote on in my PR, so we could refactor your methods to use my helper method when we merge.